doc/user-guide/How-to-build.md

```diff
@@ -59,7 +59,7 @@ $ xcode-select --install # install compilation tools
 
 Install dependencies with Brew.
 
-```
+```bash
 $ brew install erlang openssl unixodbc
```

Comment:
I'm thinking, can we delete all those `$`? When I click on the `copy to clipboard` button, it copies that dollar sign as well, which invalidates the command if I just paste it, you know what I mean 😄

Reply:
I do, will fix